pkg/util/util.go

```diff
@@ -32,6 +32,13 @@ import (
 	"k8s.io/klog"
 )
 
+const (
```

Comment:
I think this block comes from a rebase. It seems you can remove it

Reply:
Ok let me see if it breaks anything by removing it